pkg/asset/machines/ovirt/machines.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/ovirt"
 
-	ovirtprovider "github.com/openshift/cluster-api-provider-ovirt/pkg/apis/ovirtprovider/v1beta1"
+	ovirtprovider "github.com/ovirt/cluster-api-provider-ovirt/pkg/apis/ovirtprovider/v1beta1"
```

Comment:
That replacement is wrong, please revert it. Or you had to because of some go mod error?

Reply:
Go mod error, see the commit message: https://github.com/openshift/installer/pull/2745/commits/595cc3a05d1cdc6227ab5b813bfd8f2a5470bb84

The forked modules declares its import path as `github.com/ovirt/cluster-api-provider-ovirt`.
So either change that there, or here.